packages/template-retail-react-app/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -100,20 +102,29 @@ const CartSecondaryButtonGroup = ({
                     </Button>
                 </ButtonGroup>
                 <Flex alignItems="center">
-                    <Checkbox spacing={2} isReadOnly={true}>
+                    <Checkbox
+                        name={`gift-checkbox-${variant.itemId}`}
+                        spacing={2}
+                        isChecked={isAGift}
+                        onChange={(e) => {
+                            const checked = e.target.checked
+                            onIsAGiftChange(variant, checked)
+                        }}
+                    >
                         <FormattedMessage
                             defaultMessage="This is a gift."
                             id="cart_secondary_button_group.label.this_is_gift"
                         />
                     </Checkbox>
-                    <Box marginLeft={1}>
-                        <Button marginLeft={1} variant="link" size="sm">
-                            <FormattedMessage
-                                defaultMessage="Learn More"
-                                id="cart_secondary_button_group.link_learn_more"
-                            />
-                        </Button>
-                    </Box>
+                    {/* Uncomment if you want to provide a link to your gift site*/}
```

Comment:
Since we don't have link for this button, let comment it out from here to avoid a11y issue. Developer can use it when they see fit.

Reply:
Why comment out this code but delete the translation? Do we want to take a consistent action here, either delete both, or comment out both? If you choose to comment out both, you'll have to update your comment above to indicate that they can also go uncomment out the translation.

Reply:
Translations are usually automatically generated, it does not make sense to comment it there, but we can add a note here to tell dev to re-build translation if they choose to uncomment out this section.